test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.0.0-*"
```

Comment:
@eerhardt Has it exactly right here, the projects under `test` target the sharedframework in the downloaded sdk, which is why they target `1.0.0`

The projects under `TestAssets` target the latest built host/sharedfx packages and provide the test coverage. 

This split is necessary to retain the ability to run the projects under `test` outside of the build. For example, you can cd into `test` and use `dotnet test` or even the .net core Visual Studio Test runner, and get the same results as if you were running the tests in the build.